db/fixtures/miq_product_features.yml

```diff
@@ -3933,6 +3933,14 @@
         :description: Request to Provision VMs
         :feature_type: admin
         :identifier: vm_miq_request_new
+      - :name: Approve Provision VMs
+        :description: Approve Request to Provision VMs
+        :feature_type: admin
+        :identifier: vm_miq_request_approve
+      - :name: Deny Provision VMs
+        :description: Deny Request to Provision VMs
+        :feature_type: admin
+        :identifier: vm_miq_request_deny
```

Comment:
If we're really adding 2 new entitlements, spec/models/miq_product_feature_spec.rb :expected_feature_count needs to be bumped up too.